app/models/user.js

```diff
@@ -6,7 +6,12 @@ module.exports = (sequelize, DataTypes) => {
       firstName: { type: DataTypes.STRING, allowNull: false },
       lastName: { type: DataTypes.STRING, allowNull: false },
       email: { type: DataTypes.STRING, allowNull: false, unique: true },
-      password: { type: DataTypes.STRING, allowNull: false }
+      password: { type: DataTypes.STRING, allowNull: false },
+      type: {
+        type: DataTypes.STRING,
+        allowNull: false,
+        defaultValue: 'user'
+      }
```

Comment:
Just so you know, you can validate specifically for 'USER' and 'ADMIN' by adding:
`isIn: [['USER', 'ADMIN']], `
Ideally tho, their constant values should be imported from the constants file.